<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CConvolution2dParamsGuiComp</class>
 <widget class="QWidget" name="CConvolution2dParamsGuiComp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>324</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="KernelParameterLayout">
     <item>
      <widget class="QLabel" name="KernelWidthLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Kernel Width</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="KernelWidthSpin">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="KernelHeightLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Kernel Height</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="KernelHeightSpin">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="KernelTable"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
